engine/src/main/java/com/google/android/fhir/db/impl/dao/LocalChangeDao.kt

```diff
@@ -401,32 +401,37 @@ internal abstract class LocalChangeDao {
    * the updated resource in its payload, we update the payload with the reference and also update
    * the corresponding [LocalChangeResourceReferenceEntity]. We delete the original
    * [LocalChangeEntity] and create a new one with new [LocalChangeResourceReferenceEntity]s in its
-   * place. This method returns a list of the [ResourceEntity.resourceUuid] for all the resources
-   * whose [LocalChange] contained references to the oldResource
+   * place. This method returns a list of the [LocalChangeEntity.resourceUuid] for all the
+   * [LocalChange]s containing references to the oldResource.
```

Comment:
i don't think the resource uuids should be listed in square brackets actually. But also the old sentence is actually clearer about the relationship. It's the UUIDs of resources whose local changes reference the old resource.

```suggestion
   * place. This method returns a list of resource UUIDs for all the
   *  resources whose [LocalChange]s reference to the oldResource
```

Reply:
Have stuck to the same wordings but updated more of kdocs. PTAL.